src/components/ContactCard.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="contact-card__container px-4">
```

Comment:
make it a form with validation and actual ajax request.
name must have more 5 characters.
phone number must be valid 11 digit phone number (0{prefix: 2}{phone: 8})
message must have 10 characters or more.

Reply:
Added Validation